model/src/main/java/uk/gov/hmcts/payment/api/model/Payment2Repository.java

```diff
@@ -66,5 +66,5 @@ List<Tuple> findDuplicatePaymentsByDate(@Param("startDate") Date startDate,
     List<Tuple> findAllByDateCreatedBetweenAndPaymentChannel(
         @Param("fromDate") Date fromDate,
         @Param("toDate") Date toDate,
-        @Param("paymentChannel") PaymentChannel paymentChannel);
+        @Param("paymentChannel") String paymentChannel);
```

Comment:
Spring Boot 3 changed the way Hibernate manages entities and queries so can no longer pass PaymentChannel entity object directly into Hibernate query.